packages/meta/setup.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import ast
+import os
+import re
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+SOURCE_DIR = "galaxy"
+
+_version_re = re.compile(r'__version__\s+=\s+(.*)')
+
+with open('%s/project_galaxy_util.py' % SOURCE_DIR, 'rb') as f:
+    init_contents = f.read().decode('utf-8')
+
+    def get_var(var_name):
+        pattern = re.compile(r'%s\s+=\s+(.*)' % var_name)
+        match = pattern.search(init_contents).group(1)
+        return str(ast.literal_eval(match))
+
+    version = get_var("__version__")
+    PROJECT_NAME = get_var("PROJECT_NAME")
+    PROJECT_URL = get_var("PROJECT_URL")
+    PROJECT_AUTHOR = get_var("PROJECT_AUTHOR")
+    PROJECT_EMAIL = get_var("PROJECT_EMAIL")
+    PROJECT_DESCRIPTION = get_var("PROJECT_DESCRIPTION")
+
+TEST_DIR = 'tests'
+PACKAGES = []
+ENTRY_POINTS = '''
+        [console_scripts]
+        galaxy-paster=galaxy.util.pastescript.serve:run
+        galaxy-main=galaxy.main:main
+        galaxy-config=galaxy.config.script:main
+        galaxy-manage-db=galaxy.model.orm.scripts:manage_db
```

Comment:
This belong in galaxy-app after https://github.com/galaxyproject/galaxy/pull/8293 I think.

Reply:
Added to the TODO